README.md

```diff
@@ -1,48 +1,80 @@
-# rosis
-
+# ROSIS
 ROSIS (ROS2 Integration Services) is an implementation that uses [SOSS][soss] to connect ROS2 to different systems.
 
 ## Setup
+ROSIS uses a *colcon* to work.
+As dependencies, it needs a [SOSS][soss] environment with the `soss-ros2 and soss-mock` packages into it to work.
 
-This repository has a workspace already organized in order to make easy starting a connection between ROS2 and any other system.
-
-It should be downloaded with the recursive option, in order to download SOSS as a third party. That way, the initial layout will include a folder with a workspace, which contains a `src` (source) directory, and inside that last folder, the SOSS project and the ROS2 system handle will be downloaded and ready to build.
+Clone this repository into [colcon workspace][colcon], and *source* the soss dependency:
 
 ```
-$ git clone git@github.com:eProsima/ROSIS.git --recursive
+$ cd rosis_workspace
+$ git clone git@github.com:eProsima/ROS2-Integration-Service.git src/rosis
+$ source path/to/soss/setup.bash
+$ colcon build
 ```
 
-If the user wants to connect with other systems, the system handle for each system to be connected must be downloaded separately. To make things easier, this repository includes in its root directory a file called rosis.repos, to be used together with [vcstool](https://github.com/dirk-thomas/vcstool) to clone eProsima's system handles. In the root directory, execute the following command:
-
-```
-$ vcs import < rosis.repos
-```
+## Plugins packages
+ROSIS can connect ROS2 to any system that has a soss-plugin implemented.
+The official list of supported systems can be found at: [SOSS][soss].
 
-This will clone the system handles into the folder workspace/src/plugins.
+If you are a developer and you want to connect ROS2 to other systems,
+you may want to check our manual on [how to create a system handle](docs/CreatingSH.md)
 
 ## Usage
+* Into your colcon workspace, source the *colcon environment* in which ROSIS has been built (see [Setup](# Setup)):
```

Comment:
The link was not working. I think it should work with the following change, based on [this answer](https://stackoverflow.com/a/6706594):

```suggestion
* Into your colcon workspace, source the *colcon environment* in which ROSIS has been built (see [Setup][]):
```

Reply:
I did the change but still not working. It is weird. Maybe is because of the extra brackets.

Reply:
Fixed: The problem was the space between `#` and `Setup`. And the Upper case `S`.